mlflow/langchain/_langchain_autolog.py

```diff
@@ -325,7 +349,8 @@ def __init__(self, *args, **kwargs):
                 f"Failed to log inputs and outputs into `{INFERENCE_FILE_NAME}` "
                 f"file due to error {e}."
             )
-        mlflow.log_table(data_dict, INFERENCE_FILE_NAME, run_id=mlflow_callback.mlflg.run_id)
+        else:
+            mlflow.log_table(data_dict, INFERENCE_FILE_NAME, run_id=mlflow_callback.mlflg.run_id)
```

Comment:
Shouldn't this be inside `try` clause? We probably want to capture exception from the logging call.

Reply:
Yep, we should put it inside try so it doesn't block rest code execution. Thanks for catching it!